nidirect_common/nidirect_common.module

```diff
@@ -237,3 +237,18 @@ function nidirect_common_taxonomy_term_presave(EntityInterface $entity) {
     }
   }
 }
+
+/**
+ * Implements hook_form_alter().
+ */
+function nidirect_common_form_alter(&$form, FormStateInterface $form_state, $form_id) {
+  // Add description to theme/subtheme field for landing pages.
+  if (($form_id == 'node_landing_page_form') || ($form_id == 'node_landing_page_edit_form')) {
```

Comment:
I was going to suggest using route names instead of form IDs (eg: entity.node.edit_form) as they feel more natural in D8, but this is a form alter hook and the value is already provided, so I think a route based check would be less efficient.